docs/ota-client-guide/modules/ROOT/pages/deploy-checklist.adoc

```diff
@@ -16,12 +16,12 @@ Here is a checklist for all the tasks you should consider when moving to product
 |====================
 | Task  | Summary | Documentation
 |**Register the root certificate for your fleet ** |
-* If you followed our recommendations, you should have accounts for development, testing and production.
-** If you also followed our recommendation to use device-credential provisioning, you need to register your Fleet Root certificate with your production account
+* If you followed our recommendations, you should have separate environments for development, testing, and production.
+** If you also followed our recommendation to use device-credential provisioning, you need to register your Fleet Root certificate with your production environment.
```

Comment:
As with the title, I don't think this line should be edited, because using device credentials and having separate environments are both recommendations of OTA to register your root certificate.

Reply:
Yes, but does it really matter here whose recommendations users followed? It's additional info, and it makes the sentence longer (which means that users should spend more time reading it). The important things here are the condition and the action related to that condition.  :)